sql/sql_expression.cc

```diff
@@ -2019,65 +2013,6 @@ parse(ML::Parse_Context & context, bool allowUtf8)
         }
     }
 
-    //cerr << "offset = " << context.get_offset() << endl;
-    /* Three possibilities:
-       1.  (tablename).(prefix)*
-       2.  expression (AS label)
-       3.  label : expression
-    */
-    if (!matched) {
```

Comment:
That code was there to find the explicit dataset.* case before we handle the more generic case identifier*  Now we can't handle the explicit case at parsing time, so the code isnt needed anymore.